app/src/main/java/practicum/getfitla_v3/Nutrition.java

```diff
@@ -1,86 +1,210 @@
+                    counter++;
+                } catch (JSONException ex) {
+                    counter++;
+                    servingSize = row.getJSONArray("c").getJSONObject(counter).optString("v");
+                    counter++;
+                }
+                //System.out.println("Calories" + counter);
+                try {
+                    calories = row.getJSONArray("c").getJSONObject(counter).optString("v");
+                    counter++;
+                } catch (JSONException ex) {
+                    counter++;
+                    calories = row.getJSONArray("c").getJSONObject(counter).optString("v");
+                    counter++;
+                }
+
+                try {
+                    equipment = row.getJSONArray("c").getJSONObject(counter).optString("v");
+                    counter++;
+                } catch (JSONException ex) {
+                    counter++;
+                    equipment = row.getJSONArray("c").getJSONObject(counter).optString("v");
+                    counter++;
+                }
+
+
+                try {
+                    JSONObject Jingredients = (row.getJSONArray("c").getJSONObject(counter));
+                    ingredients = Jingredients.optString("v");
+                    counter++;
+                } catch (JSONException ex) {
+                    counter++;
+                    JSONObject Jingredients = (row.getJSONArray("c").getJSONObject(counter));
+                    ingredients = Jingredients.optString("v");
+                    counter++;
+                }
+
+                try {
+                    JSONObject Jprocess = (row.getJSONArray("c").getJSONObject(counter));
+                    process = Jprocess.optString("v");
+                    counter++;
+                } catch (JSONException ex) {
+                    counter++;
+                    JSONObject Jprocess = (row.getJSONArray("c").getJSONObject(counter));
+                    process = Jprocess.optString("v");
+                    counter++;
+                }
+
+                try {
+                    JSONObject Jshortdesc = (row.getJSONArray("c").getJSONObject(counter));
+                    shortdesc = Jshortdesc.optString("v");
+                } catch (JSONException ex) {
+                    counter++;
+                    JSONObject Jshortdesc = (row.getJSONArray("c").getJSONObject(counter));
+                    shortdesc = Jshortdesc.optString("v");
+                }
+
+
+                //Integer image = Integer.parseInt((row.getJSONArray("c").getString(4)));
+                image = R.drawable.nutrition; // Temporary fix until we have images
+
+                NutritionItemFormat nutrition = new NutritionItemFormat(
+                        row_id,
+                        name,
+                        shortdesc,
+                        image,
+                        // rating,
+                        //   price,
```

Comment:
I blanked out the things associated with rating and price because I had yet to add rating and prices to the google sheets and layouts. Can I keep it for now?

Reply:
I would add it to the Google Sheet as empty columns, then uncomment them here.